Pkgfile

```diff
@@ -164,9 +164,9 @@ vars:
   arm_trusted_firmware_sha512: 958cf8f9e258638e59d0fbd8b053fce7d8a9ea2fc922686c9d20ea16f79f55219ac18a12ab240c528ee98e49c2e0eef4c963fdb255cc14b92437a5b3cffc8640
 
   # renovate: datasource=github-tags extractVersion=^v(?<version>.*)$ depName=u-boot/u-boot
-  uboot_version: 2022.07
-  uboot_sha256: 92b08eb49c24da14c1adbf70a71ae8f37cc53eeb4230e859ad8b6733d13dcf5e
-  uboot_sha512: e1b3621dafac876c644ffb3d24b6e2380f5a78454c420a5b17d174e3f0b57731a95cfbff28ae7d0af4493f4bbf8d21e7f9236d61bb2414ed12295332b3d557b6
+  uboot_version: 2022.10
```

Comment:
let's skip this for now, for the jetson u-boot build there are no updated patches that work with 2022.10 of u-boot. it needs to be updated and tested manually

Reply:
K, removed uboot from the commit and bumped the tools container once more.